conf/application.conf

```diff
@@ -174,6 +174,7 @@ feature-switch {
   enable-time-machine = false
   enable-session-data-storage = true
   enable-review-and-reconcile-poa = true
+  enable-filter-coded-out-poas = true
```

Comment:
I would suspect that default should be set to false ? as we are not showing this logic at the moment

Reply:
We are using this logic at the moment, but it was supposed to be behind a feature switch so we could remove it later. We are already filtering out coded out Poas